apps/sms/views/conversation/js/compose.js

```diff
@@ -630,10 +630,9 @@ var Compose = (function() {
       var sizeExceedsLimit = this.type === 'mms' &&
         Settings.mmsSizeLimitation && this.size > Settings.mmsSizeLimitation;
 
-      // We can't send if message is empty, we're in the process of resizing or
+      // We can't send if we're in the process of resizing or
       // message is already too big.
```

Comment:
__nit:__ since we have now more space at the top comment line, please move more words there from the bottom one :)